pixeltable/catalog/table_version.py

```diff
@@ -82,6 +82,16 @@ class TableVersion:
 
     Only TableVersion and Catalog interact directly with stored metadata. Everything else needs to go through these
     two classes.
+
+    TableVersions come in three "flavors" depending on the `effective_version` and `anchor_tbl_id` settings:
+    - if both are None, it's a live table that tracks the latest version
```

Comment:
More directly: it's version is TableMd.current_version

Reply:
clarified